RELEASE.md

```diff
@@ -2,6 +2,43 @@
 
 This document guides a contributor through creating a release of JupyterLab.
 
+## The JupyterLab Release Process
+
+JupyterLab follows [semver](https://semver.org/) for the versioning of the Python and JavaScript packages.
+
+### Release Timeline
+
+Although the commitments listed below are "best effort", the JupyterLab team tries to follow a couple of guidelines:
+
+- one major version per year, which usually includes API breaking changes
```

Comment:
```suggestion
- minimum one major version per year, which usually includes API breaking changes
```

Reply:
I have been to fast to comment, I was meaning that we could have more than one per year.

Reply:
We could. But in practice this might be too fast for extension authors if they need to update their extensions each time.

Reply:
A major extension can break backwards compatibility, but is not obliged to do. I am more in favor of multiple incremental smaller upgrades, instead of a single big one. I am fine to leave it as it is and just thought useful to share my thoughts here.